Airwallex/Core/Sources/AWXAPIClient.h

```diff
@@ -49,11 +49,31 @@ NS_ASSUME_NONNULL_BEGIN
  */
 + (void)disableAnalytics;
 
+/**
+ enable analytics.
+ */
++ (void)enableAnalytics;
```

Comment:
Do we need this? It's enabled by default, and once AnalyticsLogger is inited, turning it on or off won't take effect.

Reply:
The singleton will affect all unit tests. We need to set it back so that it would not make any other tests fail.